op-chain-ops/genesis/testdata/test-deploy-config-full.json

```diff
@@ -69,6 +69,7 @@
   "faultGameMaxDepth": 63,
   "faultGameMaxDuration": 604800,
   "systemConfigStartBlock": 0,
-  "leaderElectionMaxParticipants": 5,
+  "leaderElectionNumberOfLeaders": 3,
+  "leaderElectionNumberOfSlotsPerLeader":4,
```

Comment:
This is quite small. I'm not confident that we can function with this < 3, and I'm fairly confident that we cannot function with this < 2. Setting it to 4 may be useful, in that it will increase the surface area of possible failure modes.

Reply:
The number of leaders has been set to 4 in 48a7978ba .